common/tls/src/main/java/io/helidon/common/tls/RevocationConfigBlueprint.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.tls;
+
+import java.net.URI;
+import java.util.Optional;
+
+import io.helidon.builder.api.Option;
+import io.helidon.builder.api.Prototype;
+
+@Prototype.Blueprint
+@Prototype.Configured
+interface RevocationConfigBlueprint {
+
+    /**
+     * Flag indicating whether this revocation config is enabled.
+     *
+     * @return enabled flag
+     */
+    @Option.Configured
+    @Option.DefaultBoolean(false)
+    boolean enabled();
+
+    /**
+     * Prefer CRL over OCSP.
+     * Default value is {@code false}.
+     *
+     * @return whether to prefer CRL over OCSP
```

Comment:
would be nice to have a bit of explanation of what `CRL` and `OCSP` is (ie. CRL is a link to list of revoked certificates, OCSP is an endpoint to validate specific certificate)